src/applications/caregivers/tests/config/secondaryTwo/identityInformation.unit.spec.js

```diff
@@ -0,0 +1,34 @@
+import {
+  testNumberOfErrorsOnSubmitForWebComponents,
+  testNumberOfWebComponentFields,
+} from '../helpers.spec';
+import formConfig from '../../../config/form';
+
+const {
```

Comment:
Is this destructuring just to get get the `secondaryCaregiverInformation` to get the variables to pass as params? It looks like we only use the`secondaryTwoIdentityInformation` in the line below to get the params. Would just directly getting it be clearer to read? Or does eslint not like that. Something like this: 
```const {
  title: pageTitle,
  schema,
  uiSchema,
} = formConfig.chapters.primaryCaregiverInformation.pages. secondaryTwoIdentityInformation;

Reply:
I find the destructioning cleaner to read, due to the length of the key names. It just comes down to preference, really, not an esLint rule or anything. Could be something we revisit as a group.